index.js

```diff
@@ -38,6 +38,19 @@ var argv = require('yargs')
     default: false,
     global: true
   })
+  .option('pre-release', {
+    alias: 'p',
+    describe: 'Should the release be a pre-release?',
+    type: 'boolean',
+    default: false,
+    global: true
+  })
+  .option('pre-id', {
```

Comment:
Defining a tag when publishing is the same like a git tag. pre-release and pre-id are one part of the tag.  See http://semver.org/ for more infos.